test/controllers/paye/PayeAtsMainControllerSpec.scala

```diff
@@ -49,12 +49,24 @@ class PayeAtsMainControllerSpec extends UnitSpec with GuiceOneAppPerTest with I1
 
     override val payeAuthAction: PayeAuthAction = FakePayeAuthAction
     override val payeYear = taxYear
+    override val payeAtsService = mock[PayeAtsService]
+
+    private def readJson(path: String) = {
+      val resource = getClass.getResourceAsStream(path)
+      Json.parse(Source.fromInputStream(resource).getLines().mkString)
+    }
+
+    val expectedResponse: JsValue = readJson("/paye_ats.json")
   }
 
-  "Government spend controller" should {
+  "AtsMain controller" should {
 
     "return OK response" in new TestController {
 
+
+      when(payeAtsService.getPayeATSData(eqTo(testNino), eqTo(taxYear))(any[HeaderCarrier]))
+        .thenReturn(Right(expectedResponse.as[PayeAtsData]))
```

Comment:
Consider just returning any old Right here as the result is not used, this will allow you to delete a lot of set up in this test

Reply:
agreed
changed the code to return mocked object and deleted the non required setup